### DISCUSSION TITLE
clash rule mode 会导致“出现一个未知的错误” 全页面无法加载

### DISCUSSION THREAD

**Discussion by ghost** [ywmoyue/biliuwp-lite#54] [Q&A] [answered]
关闭梯子可解决，但是web端不会出现此问题 那么请问如何对rule mode进行配置？

**Comment by ywmoyue** [comment]
![image](https://user-images.githubusercontent.com/28388300/232763641-3edfcbfe-327c-4e64-a475-f849fdad3dad.png)

![image](https://user-images.githubusercontent.com/28388300/232764101-68ec419d-dee8-4dde-b579-ec268b07a754.png)

**Comment by letsky192** [comment]
打开代理时，首页加载要等几秒。但看不到有连接。UWP Loopback 内已经排除了。

![2023-04-18_194327](https://user-images.githubusercontent.com/73059745/232768125-ea5e9305-40a9-4814-b591-c5f4ab96ff69.png)
![2023-04-18_194443](https://user-images.githubusercontent.com/73059745/232768741-cbf7f371-4a94-4647-9daa-327765e4069d.png)

**Reply by ywmoyue** [reply, marked as answer]
![image](https://user-images.githubusercontent.com/28388300/232773153-59d96dd0-145b-4a56-8dc3-6565dcba5d48.png)

不是勾上吗

**Reply by letsky192** [reply]
疏忽了理解错了意思 😣
感谢指导。
